frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -252,7 +255,25 @@ pods:
                 {{/TASKCFG_ALL_SECURITY_TRANSPORT_ENCRYPTION_ENABLED}}
                             -f ./apache-cassandra-{{CASSANDRA_VERSION}}/conf/cassandra.yaml \
                             -d ${LOCAL_SEEDS} \
+                            -u {{TASKCFG_ALL_NEW_SUPERUSER}} \
+                            -pw ${NEW_USER_PASSWORD} \
                             -p ${CASSANDRA_NATIVE_TRANSPORT_PORT} "$t" ;
+                        fi
+                {{/SECURITY_AUTHENTICATION_ENABLED}}
+                {{^SECURITY_AUTHENTICATION_ENABLED}}
+                        ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/sstableloader \
+                {{#TASKCFG_ALL_SECURITY_TRANSPORT_ENCRYPTION_ENABLED}}
+                            --keystore node.keystore \
+                            --keystore-password notsecure \
+                            --truststore node.truststore \
+                            --truststore-password notsecure \
+                            --ssl-protocol TLSv1.2 \
+                            --ssl-ciphers TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256,TLS_RSA_WITH_AES_128_CBC_SHA256,TLS_RSA_WITH_AES_128_CBC_SHA \
```

Comment:
Feels like this should be configurable. Looks like it's currently hardcoded inside of cassandra.yaml. We can hold off for a separate ticket/work since it's outside of scope but we will come back to this.